website/.husky/pre-commit

```diff
@@ -0,0 +1,2 @@
+cd website
+npx lint-staged
```

Comment:
```suggestion
pnpm lint-staged
```
Since our package manager for this project is pnpm, we can use it in the lint.
This also ensures that everyone is using the same version of pnpm (because of the packagemanager field in package.json)